src/baseline_bots/bots/smart_order_accepter_bot.py

```diff
@@ -495,6 +495,8 @@ def __call__(self, rcvd_messages: List[Tuple[int, Message]]):
         # avoid get_stance in the first phase of game
         if self.game.get_current_phase()!='S1901M':
             self.stance.get_stance()
+            
+        powers = self.stance.stance[self.power_name]
```

Comment:
Could you use `utils.get_other_powers()` instead? It would be an easy change

Reply:
Sure